framework/include/eventloop/eventloop.h

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/**
+ * @defgroup Eventloop Eventloop
+ * @ingroup Eventloop
+ * @brief Provides APIs for Eventloop
+ * @{
+ */
+/**
+ * @file eventloop/eventloop.h
+ */
+#ifndef __EVENTLOOP_H__
+#define __EVENTLOOP_H__
+
+#include <libtuv/uv.h>
+
+/**
+ * @brief Event Loop Timer structure
+ */
+typedef uv_timer_t event_timer;
```

Comment:
since your are using libuv's function directly, I think more typedef not neccessary.